raiden-dapp/package.json

```diff
@@ -24,13 +24,16 @@
   },
   "dependencies": {
     "compare-versions": "^3.6.0",
+    "cordova-sqlite-storage": "^6.0.0",
     "core-js": "^3.8.3",
     "ethereum-blockies-base64": "^1.0.2",
     "ethers": "^5.0.31",
     "fontsource-roboto": "^4.0.0",
     "idb": "^6.0.0",
     "lodash": "^4.17.20",
     "loglevel": "^1.7.1",
+    "pouchdb": "^7.2.2",
```

Comment:
Any help why this is necessary? Shouldn't this be included from the SDK?

But without this I got:
```
error: 'pouchdb' should be listed in the project's dependencies. Run 'npm i -S pouchdb' to add it (import/no-extraneous-dependencies) at src/services/raiden-service.ts:3:21:
  1 | import type { BigNumber, BigNumberish, providers } from 'ethers';
  2 | import { constants, utils } from 'ethers';
> 3 | import PouchDB from 'pouchdb';
```

Reply:
If you import this directly in the dApp, then yes you should follow this. You should not rely on sub-dependencies of you dependencies. They can change and then your package fails suddenly. It is a dependency, so that's it.